### DISCUSSION TITLE
[Feedback] (rework inside) The new Alien enemies are great... the ruins not so much

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#6867] [Unstable feedback]
The ruins overhaul has introduced several new gameplay elements that weren't present before:

-Guardians, a new, very scary type of enemy
-Vents which can either spawn burning gases or feeders
-Alien terminals, which can be used to send signals

While the guardians are great with their unique AI and abilities, the ruins themselves are rather simplistic. 

The alien terminals would suggest some sort of puzzle with their multiple inputs but in reality they're only used for simple, one-time actions, with no failure state on the wrong input. It boils down to:
_Press red button = blow up pod
Press green button = open door_

As players learn the layout of each ruin room, they'll shut down systems and traps with ease.

The ruins also have *a lot* of doors, which makes it somewhat tedious to navigate. The purpose of doors might be to slow down players as they're chased by Guardians, but it also slows them down when they aren't. It makes navigating ruins somewhat more annoying, but not more difficult.

Overall the ruins themselves don't have that much replayability with most layouts being somewhat simplistic. 

**Here I've remade every single ruin room as a proof of concept with more puzzles and interactive elements**
_(note: **this was made before the most recent unstable build**. it doesn't have lights or vents, and the blue/yellow buttons have been removed from terminals which can break the 4-colour puzzles)_
Thanks to Workingjoe for making it free of sync issues in multiplayer

**How to test them:**
Replace all the .sub files (after making a backup) in the RuinModules folder
![image](https://user-images.githubusercontent.com/73229309/136586587-6ae8440b-9c7c-4bf1-b075-c18861a12afa.png)

[NewAlienRuins.zip](https://github.com/Regalis11/Barotrauma/files/7312607/NewAlienRuins.zip)

Snippet of one room with commentary
![image](https://user-images.githubusercontent.com/73229309/136588487-95bd3000-90fb-4dcf-8a5b-34f766ed3ce6.png)
 
 These new ruins aim to be less tedious, more dynamic and interactive with:
 -Failure states on alien terminals, inputting the wrong button has negative consequences
 -Much less doors and motion sensors
 -Door traps to cut off fleeing divers when chased by a guardian
 -Colour-based types of puzzles with hidden colored "clues" in the room
 -Coordinated button pressing to open doors or shut down mechanisms
 And some more....

**Comment by LePiromano** [comment]
i agree
